### PR TITLE
Dedupe service Terraform config

### PR DESCRIPTION
**Pull request by ldbnr4**

### CODE REVIEW

File: config/variables.tf [hunk 1]
```diff
@@ -108,6 +108,32 @@ variable "gcs_to_bq_runner_role_id" {
   type        = string
 }
 
+# Dedupe Cloud Run Service Vars
+variable "dedupe_service_name" {
+  description = "Name of the Cloud Run service for removing "
```
Comment (mayaspivak) [inline config/variables.tf:113]: Finish comment here and update comments below to dedupe service instead of GCS-to-BQ service.

**Review by mayaspivak — CHANGES_REQUESTED** [review]

File: config/variables.tf (hunk 1 above)
Comment (ldbnr4) [reply config/variables.tf]: Ah thanks for catching that.

File: config/test/test.tfvars [hunk 2]
```diff
@@ -5,3 +5,7 @@ bq_dataset_name                        = "test_data_sources"
 bq_manual_dataset_name                 = "test_manual_data_sources"
 bq_cdc_covid_deaths_dataset_name       = "cdc_covid_deaths"
 export_bucket                          = "het-data-tables"
+dedupe_service_name                    = "dedupe-service"
```
[the remark this answers is not in the document]
Comment (ldbnr4) [reply config/test/test.tfvars]: I see, done.

File: .github/workflows/deployToTest.yml [hunk 3]
```diff
@@ -48,6 +48,27 @@ jobs:
           dockerfile: 'run_gcs_to_bq/Dockerfile'
           image-path: 'gcr.io/${{ secrets.TEST_PROJECT_ID }}/gcs-to-bq'
     
+  build-dedupe:
```
[the remark this answers is not in the document]
Comment (ldbnr4) [reply .github/workflows/deployToTest.yml:51]: Thanks for that insight. Added you as a reviewer for that change in the other repo

**Pull request merged**